tests/page-objects/apdex/page.js

```diff
@@ -268,60 +199,8 @@ module.exports = class Page {
       });
   }
 
-  async syncData () {
-    console.log(`TIME IS::: ${await driver.getDeviceTime()}`);
-    //change date here - WIP
-    await browser.pause(5000);
-    await this.clickDisplayedElem(this.tabDropdown);
-    await this.clickDisplayedElem(this.menuItemSyncNow);
-    await browser.pause(1000);
-    await this.tabDropdown.click();
+  async getButton(label) {
+    return $('//*[@text="' + label +'"]');
   }
```

Comment:
Because not using `await` not need of `async` 
```suggestion
 getButton(label) {
    return $('//*[@text="' + label +'"]');
  }
```